client/app/dbaas/metrics/dashboard/metrics-dashboard.controller.js

```diff
@@ -0,0 +1,121 @@
+(() => {
+    class MetricsDashboardCtrl {
+        constructor ($scope, $stateParams, $q, $translate, CloudMessage, ControllerHelper, MetricService, METRICS_ENDPOINTS, RegionService, SidebarMenu) {
+            this.$scope = $scope;
+            this.$stateParams = $stateParams;
+            this.$q = $q;
+            this.$translate = $translate;
+            this.serviceName = $stateParams.serviceName;
+            this.ControllerHelper = ControllerHelper;
+            this.CloudMessage = CloudMessage;
+            this.MetricService = MetricService;
+            this.graphs = METRICS_ENDPOINTS.graphs;
+            this.RegionService = RegionService;
+            this.SidebarMenu = SidebarMenu;
+
+            this.loading = {};
+            this.usage = {};
+            this.configuration = {};
+            this.plan = {};
+        }
+
+        $onInit () {
+            this.loading.service = true;
+            this.loading.consumption = true;
+            this.initTiles();
+        }
+
+        initTiles () {
+            this.MetricService.getService(this.serviceName)
+                .then(service => {
+                    this.usage.quota = {
+                        mads: service.data.quota.mads,
+                        ddp: service.data.quota.ddp
+                    };
+                    this.configuration = {
+                        name: service.data.name,
+                        description: service.data.description,
+                        retention: service.data.quota.retention,
+                        datacenter: this.transformRegion(service.data.region.name)
+                    };
+                    this.plan.offer = service.data.offer;
+                })
+                .finally(() => {
+                    this.loading.service = false;
+                });
+            this.MetricService.getConsumption(this.serviceName)
+                .then(cons => {
+                    this.usage.conso = { mads: cons.data.mads, ddp: cons.data.ddp };
+                    this.initMessages();
+                })
+                .finally(() => {
+                    this.loading.consumption = false;
+                });
+
+        }
+
+        initMessages () {
+            if (this.computeUsage(this.usage.conso.mads, this.usage.quota.mads) > 70) {
```

Comment:
Done. I defined this.limit.warning and this.limit.danger in order to respect the ui-kit definition.